web/search.dart

```diff
@@ -203,24 +219,55 @@ void initializeSearch(
         query, "<strong class='tt-highlight'>$sanitizedText</strong>");
   }
 
+  Map<String,Element> categoriesMap = new LinkedHashMap();
```

Comment:
We prefer to use collection literals. Are you picking LinkedHashMap for a specific reason?

https://dart.dev/guides/language/effective-dart/usage#do-use-collection-literals-when-possible

Reply:
Addressed in new commit